scripts/check-manual-tests.sh

```diff
@@ -26,6 +27,11 @@ then
   MANUAL_TEST_SERVER_OPTIONS="$MANUAL_TEST_SERVER_OPTIONS -i $IDENTITY_FILE"
 fi
 
+if [ ! -z "$TS_CONFIG" ]
+then
+  MANUAL_TEST_SERVER_OPTIONS="$MANUAL_TEST_SERVER_OPTIONS --tsconfig $TS_CONFIG"
+fi
```

Comment:
I would add `--tsconfig ./tsconfig.test.json` to L43 and remote support for the `-t` flag. This script is used only for verification on CI and when releasing stuff. We do not have customize it.